core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.spec.ts

```diff
@@ -78,9 +78,17 @@ describe('Edit Thumbnail Modal Component', () => {
   }
 
   let mockSvgSanitizerService = {
-    getInvalidSvgTagsAndAttrsFromDataUri: (dataUri: string) => {
-      return { tags: ['script'], attrs: [] };
+    parseDataURI: (dataURI: string) => {
+      return Document;
```

Comment:
Is this correct? I'm not sure why parsing a data URI would result in a Document; does the function name need to be changed?

Similarly below, I'm not sure why an "svg" is a Document. In JavaScript, isn't Document supposed to be an html document?

Reply:
Ah, makes sense (re second paragraph). Maybe adding a comment with a link will help future devs who are looking at it.

Reply:
Done!